### ISSUE TITLE
Recurrent gingivitis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7536]
**Preferred term label:**
Recurrent gingivitis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
619407 
PMID: 34854278  4/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
